app/command_test.go

```diff
@@ -79,23 +79,31 @@ func TestExecuteCommand(t *testing.T) {
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
-	args := &model.CommandArgs{
-		Command: "/code happy path",
+	// Testing slash command message is correctly parsed
```

Comment:
Can we add a test for a trigger word missing the `/`?

Reply:
yes